src/index.html

```diff
@@ -6,8 +6,45 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="header" data-qa="card">
+      <div class="container">
+        <div class="container__image">
+          <img src="./images/imac.jpeg"
+          alt="Apple iMac 27-inch Retina 5K Monoblock"
+          class="image"
+          >
+        </div>
+        <h1 class="container__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
```

Comment:
```suggestion
        <h2 class="container__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
```
We expect to have a lot of cards on the page